source/connect.txt

```diff
@@ -16,6 +16,15 @@ This page outlines how to use |compass| to connect to a
 MongoDB host. You can connect to a standalone, replica set,
 or sharded cluster host.
 
+.. admonition:: MongoDB Atlas
+   :class: note
+
+   If you need to create a MongoDB host, consider using 
+   `MongoDB Atlas <https://www.mongodb.com/cloud/atlas?tck=docs_compass>`__.
+   Atlas is a cloud-hosted database-as-a-service which requires no 
+   installation, offers a free tier to get started, and provides a 
+   copyable connection string for |compass-short|.
```

Comment:
Wondering if we can maybe tweak this a bit to explain the last item a bit more, like maybe "...provides a copyable URI to easily connect Compass to your MongoDB deployment.", 

or if you want fewer words, you could even just say "provides a copyable URI to easily connect to your deployment"

or some combination of those.

Reply:
"... provides a copyable URI to easily connect |compass-short| to your deployment."